scripts/simplesamlphp/acquia_config.php

```diff
@@ -84,7 +76,14 @@
 
 }
 elseif (getenv('AH_SITE_ENVIRONMENT')) {
-  // Set  ACE ad ACSF sites based on hosting database and site name.
+  /**
+   * Support multi-site and single site installations at different base URLs.
+   *
+   * Overide $config['baseurlpath'] = "https://{yourdomain}/simplesaml/"
+   * to customize the default Acquia configuration.
+   */
+  $config['baseurlpath'] = $protocol . $_SERVER['HTTP_HOST'] . $port . '/simplesaml/';
```

Comment:
Noting here that `$protocol` and `$port` are not defined unless the code block above is uncommented. This was also true of the code prior to this PR.